### PR TITLE
Fix formatting in querier component doc

### PR DESCRIPTION
**Pull request by jhalterman**
#### What this PR does

Fixes the formatting in the querier component doc to have two top level bullet points to describe -querier.query-ingesters-within usage. 

Before the change, docs render as:

Screenshot 2024-01-08 at 8 21 40 PM

After the change, docs render as:

Screenshot 2024-01-08 at 8 21 20 PM

#### Which issue(s) this PR fixes or relates to

Fixes #<issue number>

#### Checklist

- [ ] Tests updated.
- [x] Documentation added.
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`.
- [ ] [`about-versioning.md`](/docs/sources/mimir/configure/about-versioning.md) updated with experimental features.

### CODE REVIEW

File: docs/sources/mimir/references/architecture/components/querier.md [hunk 1]
```diff
@@ -46,8 +46,8 @@ The request to the ingesters fetches samples that have not yet been uploaded to
 
 The configured period for `-querier.query-ingesters-within` should be:
 
-- greater than `-querier.query-store-after` and,
```
Comment (jhesketh) [inline docs/sources/mimir/references/architecture/components/querier.md:49]: personally I think we should be keeping the `and,`. Other than that, lgtm.

Comment (jhalterman) [reply docs/sources/mimir/references/architecture/components/querier.md:49]: Yea, it's a bit subjective, but usually sets of bullet points, including elsewhere in our docs, imply `and`.

Comment (charleskorn) [reply docs/sources/mimir/references/architecture/components/querier.md:49]: Perhaps it'd be clearer if the sentence above the dot points was something like "The configured period for `-querier.query-ingesters-within` should be the greater of both:"?

**Review by charleskorn — APPROVED** [review]
LGTM

File: docs/sources/mimir/references/architecture/components/querier.md (hunk 1 above)
Comment (jhalterman) [reply docs/sources/mimir/references/architecture/components/querier.md:49]: Does `should be greater than both:` work the same for you?

Comment (charleskorn) [reply docs/sources/mimir/references/architecture/components/querier.md:49]: Yep, sounds good to me.

**Pull request merged**